server/api/crud/model_monitoring/deployment.py

```diff
@@ -587,6 +591,27 @@ def is_monitoring_stream_has_the_new_stream_trigger(self) -> bool:
             return False
         return True
 
+    @staticmethod
+    def _create_tsdb_application_tables(project: str, **kwargs):
+        """Each project writer service writes the application results into a single TSDB table and therefore the
+        target table is created during the writer deployment"""
+
+        if (
+            mlrun.mlconf.model_endpoint_monitoring.tsdb_connector_type
+            == mm_constants.TSDBTarget.V3IO_TSDB
+        ):
+            tsdb_configurations = {
+                "access_key": kwargs["access_key"],
+            }
```

Comment:
Because you don't need special access key this check is redundant, and you use  `get_tsdb_connector()` without extra kwargs.